lib/content-services/src/lib/tag/tags-creator/tags-creator.component.spec.ts

```diff
@@ -382,6 +382,14 @@ describe('TagsCreatorComponent', () => {
                 expect(getFirstError()).toBe('TAG.TAGS_CREATOR.ERRORS.ALREADY_ADDED_TAG');
             }));
 
+            it('should show error for prohibited characters', fakeAsync(() => {
```

Comment:
Is there a need to wrap these tests in fakeAsync() ?? It's there for almost all the tests in the file though

Reply:
yes, we need 'fakeAsync' for typeTag()

Reply:
oh, ok. Got it. Thanks